### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by skhadriatl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,4 @@ test
 what change? yes edited
 this change
 
-another new line here - edit here
```
Comment (skhadriatl) [inline README.md:10]: what's this?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,4 @@ test
...
-
-and third line here - also edit here
+another new line here - edit her
```
Comment (skhadriatl) [inline README.md:8]: another one?

File: README.md [hunk 3]
```diff
@@ -7,6 +7,4 @@ test
 what change? yes edited
```
Comment (skhadriatl) [inline README.md]: no change.

File: README.md (hunk 1 above)
Comment (wliu222) [reply README.md:10]: good change

**Pull request closed without merging**